app/src/main/java/edu/uw/tcss450/chatapp_group1/ui/weather/WeatherListViewModel.java

```diff
@@ -41,8 +40,8 @@ public WeatherListViewModel(@NonNull Application application) {
         mToggleZipcode = new MutableLiveData<>();
         mToggleZipcode.setValue(false);
         mCurrentZipcode = new MutableLiveData<>();
-        mCurrentZipcode.setValue("98030");
-        updateZipcode("98030");
+        mCurrentZipcode.setValue("98403");
```

Comment:
I thought you were getting the zipcode from the device location and not hardcoding it?

Reply:
yes. it does update the location right away. this one is a default zipcode incase when the user denies the request access location